core/src/main/java/org/frankframework/pipes/PasswordGeneratorPipe.java

```diff
@@ -31,75 +32,62 @@
 
 
 /**
- * Returns random password.
+ * Generates a random password.
  *
- * @author  Milan Tomc
- * @since   4.5
+ * @author Milan Tomc
+ * @since 4.5
  */
 @ElementType(ElementTypes.TRANSLATOR)
 public class PasswordGeneratorPipe extends FixedForwardPipe {
 
-	private String lCharacters="abcdefghijklmnopqrstuvwxyz";
-	private String uCharacters="ABCDEFGHIJKLMNOPQRSTUVWXYZ";
-	private String numbers="0123456789";
-	private String signs=";:_%$#@!><";
+	private String lCharacters = "abcdefghijklmnopqrstuvwxyz";
+	private String uCharacters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+	private String numbers = "0123456789";
+	private String signs = ";:_%$#@!><";
 
 	private SecureRandom random;
-	private boolean useSecureRandom = true; // more secure but much slower
 
-	int numOfLCharacters=2;
-	int numOfUCharacters=2;
-	int numOfDigits=2;
-	int numOfSigns=2;
+	int numOfLCharacters = 2;
+	int numOfUCharacters = 2;
+	int numOfDigits = 2;
+	int numOfSigns = 2;
 
 	@Override
 	public void configure() throws ConfigurationException {
 		super.configure();
 
-		if (useSecureRandom){
-			try {
-				random= SecureRandom.getInstance("SHA1PRNG");
-			} catch (NoSuchAlgorithmException e) {
-				try{
-					random= SecureRandom.getInstance("IBMSecureRandom");
-				}
-				catch (NoSuchAlgorithmException ex){
-					throw new ConfigurationException("PasswordGeneratorPipe: ", ex);
-				}
-			}
+		try {
+			random = SecureRandom.getInstance("SHA1PRNG");
```

Comment:
Kunnen we dit een static field maken? We hebben toch maar 1 instance nodig voor alle pwd generator pipes, lijkt me?
(SecureRandom is thread-safe)

Reply:
Eens; ik pas het aan.